subcommand/server-acl-init/command_ent_test.go

```diff
@@ -260,9 +294,25 @@ func TestRun_ACLPolicyUpdates(t *testing.T) {
 			require.Equal(0, responseCode, ui.ErrorWriter.String())
 
 			// Check that the policies have all been updated.
+			secondRunExpectedPolicies := []string{
+				"dns-policy",
+				"client-token",
+				"catalog-sync-token",
+				"connect-inject-token",
+				"mesh-gateway-token",
+				"client-snapshot-agent-token",
+				"enterprise-license-token",
+				"cross-namespace-policy",
+			}
 			policies, _, err = consul.ACL().PolicyList(nil)
 			require.NoError(err)
 
+			// Check that we the right number of policies. The actual
```

Comment:
```suggestion
			// Check that we attach the right number of policies. The actual
```

Reply:
Went with "have" instead, since only the `cross-namespace-policy` is attached to the namespace.